sdks/python/apache_beam/ml/inference/pytorch_inference_test.py

```diff
@@ -237,10 +237,8 @@ def test_run_inference_kwargs_prediction_params(self):
     inference_runner = TestPytorchModelHandlerForInferenceOnly(
         torch.device('cpu'))
     predictions = inference_runner.run_inference(
-        batch=KWARGS_TORCH_EXAMPLES,
-        model=model,
-        prediction_params=prediction_params)
-    for actual, expected in zip(predictions, KWARGS_TORCH_PREDICTIONS):
+        batch=KEYED_TORCH_EXAMPLES, model=model, extra_kwargs=extra_kwargs)
```

Comment:
I think we should keep these args as anonymous

Reply:
Can you clarify by anonymous args?

Reply:
ok I've been thinking about this:

I was playing around with ways to make arguments specific to run_inference and I think there are only three ways. Either what you have done, anonymous args, or an if statement

if inference_args:
  model_handler.run_inference(model, batch, inference_args)
else:
  model_handler.run_inference(model, batch)

I'm not sure what I prefer now that I'm looking at it.


The if statement has the advantage of allowing clients that don't expect this argument to fail or pass without modifcations.